app/controllers/verify/sessions_controller.rb

```diff
@@ -20,6 +19,18 @@ def new
     end
 
     def create
+      result = idv_form.submit(profile_params)
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED_FORM, result.to_h)
+
+      if result.success?
+        submit_idv_job
+        redirect_to verify_session_result_path
+      else
+        process_failure
+      end
+    end
+
+    def show
```

Comment:
I'll have to think about this some more, but one thing we can say for sure is that this `show` method is different from all the other `show` methods we currently have in that it is acting a lot like a `create` method since it's processing a form and acting on that submission as opposed to just rendering a view.

As for the vendor limits, my understanding was that the limits were to prevent too many failed attempts by a user. If the vendor is only temporarily down for 5 minutes, users will be prevented from trying again for another 24 hours if we count the attempts against them. Perhaps we should open up this discussion to the rest of the team? I'm not sure what would be the ideal way to handle this.